CHANGELOG.md

```diff
@@ -1,103 +1,114 @@
-### 1.14.11
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## Unreleased
+
+<!-- New PRs should document their changes here. -->
+
+### [1.14.11] - 2016-04-11
 - Radically simpler inlining design. Move all imports and scripts into document
   body, replace imports inline.
 - Add tests for more complicated inlining scenarios
 
-### 1.14.10
+### [1.14.10] - 2016-04-07
```

Comment:
It's the keep a changelog convention. I don't have a strong opinion, I just want a convention.